### ISSUE TITLE
Resolve DB Migration Issues (Also fixes test runner)

### ISSUE THREAD

**Issue by ryaustin** [kjcioffi/django-ecommerce#7]
Given the current state of the db due to the phonenumber field class I recommend blowing away your migrations and rebuilding your DB. Optional, but I would also move to sqlite for this phase of the project. Here are the steps to take:

- [ ] Delete all migrations or move to new folder
- [ ] modify DATABASES setting in settings.py for store app to:
```DATABASES = {
    "default": {
        "ENGINE": "django.db.backends.sqlite3",
        "NAME": BASE_DIR / "db.sqlite3",
    }
}
```

- [ ] Make migrations `python manage.py makemigrations`
- [ ] migrate `python manage.py migrate`
- [ ] finally run your tests and verify that there are no db or migration errors. You will have some failed tests due to phone number field validation that you should resolve as needed.

**Comment by kjcioffi** [comment]
I used an alternative approach which I think you'll like more, @ryaustin. I decided to go against deleting all my migrations. 

Resolution is in #8

**Comment by ryaustin** [comment]
as mentioned, resolved in #8
